api/catalog/settings.py

```diff
@@ -260,6 +260,9 @@ def _make_cache_config(dbnum: int, **overrides) -> dict:
         "USER": config("DJANGO_DATABASE_USER", default="deploy"),
         "PASSWORD": config("DJANGO_DATABASE_PASSWORD", default="deploy"),
         "NAME": config("DJANGO_DATABASE_NAME", default="openledger"),
+        "OPTIONS": {
+            config("DJANGO_DATABASE_APPLICATION_NAME", default="openverse-api"),
+        },
```

Comment:
Is `OPTIONS` really meant to be a set? Is a key missing?

Reply:
Oh good catch

Reply:
GAH, fixed. Thank you.

Reply:
It's a [dictionary](https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-OPTIONS) but I can't more specific information for Postgres yet.